### PR TITLE
Squash CI Bugs

### PR DESCRIPTION
**Pull request by jdamiba**
closes https://github.com/plotly/plotly.r-docs/issues/6

The purpose of this PR is to squash bugs which are preventing the `master` branch from passing CI. 

This involves:

- [x] adding `chopper.ply` and `2085.csv` data files to the repo filesystem instead of making API calls at build time

- [x] adding `random` package to `DESCRIPTION`

CI build steps renamed for clarity.

### CODE REVIEW

File: r/2017-01-19-buttons.Rmd [hunk 1]
```diff
@@ -343,8 +343,8 @@ library(plotly)
 # Sys.setenv('MAPBOX_TOKEN' = 'your mapbox token')
 
 # read in wind turbines and farms data
-df_wind = read.csv('https://plot.ly/~datasets/2805.csv')
-df_farms = read.csv('https://plot.ly/~jackp/17256.csv')
+df_wind = read.csv('data/2805.csv')
```
Comment (nicolaskruchten) [inline r/2017-01-19-buttons.Rmd]: Why must we in-line this one and not the next one? Seems odd

File: r/2016-06-17-3d-tri-surf.Rmd [hunk 2]
```diff
@@ -73,12 +73,7 @@ p
 library(plotly)
 library(geomorph)
 
-plyFile <- 'http://people.sc.fsu.edu/~jburkardt/data/ply/chopper.ply'
-dest <- basename(plyFile)
-if (!file.exists(dest)) {
-  download.file(plyFile, dest)
-}
-
+dest <- "data/chopper.ply"
```
Comment (nicolaskruchten) [inline r/2016-06-17-3d-tri-surf.Rmd]: Any idea why this must be inlined and no other files? Just the download.file command doesn’t work?

File: r/2017-01-19-buttons.Rmd (hunk 1 above)
Comment (jdamiba) [reply r/2017-01-19-buttons.Rmd]: The output of the command `df_wind = read.csv('https://plot.ly/~datasets/2805.csv')` is:

```
df_wind = read.csv('https://plot.ly/~datasets/2805.csv')

 *** caught segfault ***
address 0x10b966000, cause 'memory not mapped'

Traceback:
 1: scan(file = file, what = what, sep = sep, quote = quote, dec = dec,     nmax = nrows, skip = 0, na.strings = na.strings, quiet = TRUE,     fill = fill, strip.white = strip.white, blank.lines.skip = blank.lines.skip,     multi.line = FALSE, comment.char = comment.char, allowEscapes = allowEscapes,     flush = flush, encoding = encoding, skipNul = skipNul)
 2: read.table(file = file, header = header, sep = sep, quote = quote,     dec = dec, fill = fill, comment.char = comment.char, ...)
 3: read.csv("https://plot.ly/~datasets/2805.csv")
```

Inlining solves the error, which does not appear for `df_farms`.

Comment (nicolaskruchten) [reply r/2017-01-19-buttons.Rmd]: Weird

**Comment by nicolaskruchten** [comment]
💃

File: r/2016-06-17-3d-tri-surf.Rmd (hunk 2 above)
Comment (nicolaskruchten) [reply r/2016-06-17-3d-tri-surf.Rmd]: I’m fine with inlining, just curious about when we must vs can just continue as was

Comment (jdamiba) [reply r/2016-06-17-3d-tri-surf.Rmd]: When trying to access the file at the URL, the following error is thrown:

```
trying URL 'http://people.sc.fsu.edu/~jburkardt/data/ply/chopper.ply'
Quitting from lines 73-106 (2016-06-17-3d-tri-surf.Rmd) 
Error in download.file(plyFile, dest) : 
  cannot open URL 'http://people.sc.fsu.edu/~jburkardt/data/ply/chopper.ply'
Calls: <Anonymous> ... withCallingHandlers -> withVisible -> eval -> eval -> download.file
In addition: Warning messages:
1: In rgl.init(initValue, onlyNULL) : RGL: unable to open X11 display
2: 'rgl_init' failed, running with rgl.useNULL = TRUE 
3: In download.file(plyFile, dest) :
  URL 'https://people.sc.fsu.edu/~jburkardt/data/ply/chopper.ply': status was 'SSL connect error'
```

I believe SSL is set up correctly in the container, as `libcurl4-openssl-dev` and `ca-certificates` are both installed. Seems like an issue with the server?

**Pull request merged**